### DISCUSSION TITLE
Is it possible to move media between models and retain the conversions as well?

### DISCUSSION THREAD

**Discussion by shealan** [spatie/laravel-medialibrary#2163] [General] [answered]
Hey all, loving this package so far!

When I upload a temp file I create a basic preview image.

When I save a new model I would then like to move that temp media across to that model, keeping the preview conversion to avoid having to generate it again. I'd then like to queue some additional conversions which are defined on that new model.

Is this possible? The "move" method and "copy" method both seem relevant here but I suppose I am actually looking for a kind of hybrid.

Does the copy method actually retain the conversions? The documentation is a little light on that page.

**Comment by freekmurze** [comment, marked as answer]
When moving or copying media from one model to the other, all conversions are performed again because the destination model may have defined other conversions defined as the source model.
